includes/scenes/AScene.hpp

```diff
@@ -6,12 +6,12 @@
 
 class AScene {
 	public:
-		explicit AScene(Gui * gui);
+		AScene(Gui * gui, float const &dtTime);
```

Comment:
pourquoi t'as viré le keyword explicit ici ?

Reply:
je le met que quand il y a un seul arg

Reply:
sinon ça forcerais à donner un float, on pourais pas donner un double avec conversion implicite

Reply:
ok

Reply:
en general la pratique courante c'est de mettre explicit seulement sur les constructeurs qui prennent un seul param

```"As discussed in this GFact, in C++, if a class has a constructor which can be called with a single argument, then this constructor becomes conversion constructor because such a constructor allows conversion of the single argument to the class being constructed.
We can avoid such implicit conversions as these may lead to unexpected results. We can make the constructor explicit with the help of explicit keyword. For example, if we try the following program that uses explicit keyword with constructor, we get compilation error."
```
> geeksforgeeks.org/g-fact-93

Reply:
Pour éviter des soucis avec le constructeur pas copy, non ?